pages/osx/sips.md

```diff
@@ -0,0 +1,28 @@
+# sips
+
+> Apple Scriptable Image Processing System.
+> Raster/Query images and ColorSync ICC Profiles.
+
+- Specify an output directory so originals does not get modified:
+
+`sips --out /path/to/out ...`
```

Comment:
path/to/out should be in tokens `{{}}`. Also, what are the `...` for ?

Reply:
I will fix that, the dots were meant to imply the rest of the users command (appending to another command). Should i change it to {{rest_of_command}} or just remove it?